theano/tensor/nnet/abstract_conv.py

```diff
@@ -1622,7 +1635,7 @@ def perform(self, node, inp, out_):
             img = new_img
         if not self.filter_flip:
             kern = kern[(slice(None), slice(None)) + (slice(None, None, -1),) * self.convdim]
-        conv_out = self.conv(img, kern, mode="valid", dilation=self.filter_dilation)
+        conv_out = self.conv(img, kern, mode="valid", dilation=self.filter_dilation, num_groups=self.num_groups)
         conv_out = conv_out[(slice(None), slice(None)) +
                             tuple(slice(None, None, self.subsample[i])
                                   for i in range(self.convdim))]
```

Comment:
The method R_op bellow will need update. Do that later when the the rest work.

But make it raise an error now if groups isn't 1.